src/python/pants/backend/python/goals/pytest_runner.py

```diff
@@ -60,6 +66,7 @@ class PythonTestFieldSet(TestFieldSet):
 
     sources: PythonTestsSources
     timeout: PythonTestsTimeout
+    runtime_binary_dependencies: PythonRuntimeBinaryDependencies
```

Comment:
This is extra nice because it means that currently only python_tests targets can have these deps. If we want to add them to other target types we can do that in the future once we figure out what that might mean. It may mean that we want to find all runtime_binary_dependencies on the transitive closure (via regular `dependencies`), but we can worry about that later.

Reply:
Exactly! Target authors have a new tool they can optionally choose to use, without being forced to implement certain semantics that don't make sense for them.